lib/src/zap/sock/zap_sock.c

```diff
@@ -113,7 +113,8 @@ static zap_err_t __sock_send_msg(struct z_sock_ep *sep, struct sock_msg_hdr *m,
 static zap_err_t __sock_send_msg_nolock(struct z_sock_ep *sep,
 					struct sock_msg_hdr *m,
 					size_t msg_size,
-					const char *data, size_t data_len);
+					const char *data, size_t data_len,
+					struct z_sock_io *io);
```

Comment:
@narategithub `struct z_sock_io *io` isn't used in the function. I want to remove it from the signature if I didn't miss anything.